ui/partials/perf/comparechooserctrl.html

```diff
@@ -21,11 +26,16 @@
         <div class="panel-heading">New</div>
         <div class="panel-body">
           <label for="new-project-selector">Project</label>
-          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
+          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects" ng-change="updateNewRevisionTips()"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group" ng-class="{'has-error': newRevisionError}">
-            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
-            <br/>
+            <input id="new-revision-input" maxlength="12" class="form-control" type="text" ng-model="newRevision"/>
+            <div class="input-group-btn">
+              <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Pushes<span class="caret"></span></button>
+              <ul class="dropdown-menu dropdown-menu-right" >
+                <li ng-repeat="tip in newTipList"><a ng-click="getNewTipRevision(tip.revision)">{{tip.revision}} {{tip.author}}</a></li>
```

Comment:
Similarly here, you might wish to next the anchor on a new line, inside the `li`.

I know a lot of the file is like this though.